queries.sql

```diff
@@ -45,3 +45,19 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) from animals WHERE date_of_birth BETWEEN '1990-01-01' and '2000-01-01' GROUP BY species;
+
+SELECT name FROM animals JOIN owners ON owners.full_name = 'Melody Pond' AND owners.id = animals.owner_id;
+SELECT animals.name FROM animals JOIN species ON species.name = 'Pokemon' AND animals.species_id = species.id;
+SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON owners.id=animals.owner_id ORDER BY full_name;
+SELECT species.name, COUNT(animals.name) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.id;
```

Comment:
- [ ] Great work writing the command that gives the accurate result. Your command is accurate but the screenshot you attached displays the wrong result for this query. Kindly update your screenshot to reflect the accurate result.

### Result of Query

![image](https://user-images.githubusercontent.com/77106962/206312581-cb8fa55e-2f65-4a19-bed7-6fd4fd19cc40.png)

### Screenshot Attached

![image](https://user-images.githubusercontent.com/77106962/206312664-82155c76-83e6-449c-b4bb-999fdd4dc78c.png)

#### Tip: There are `5 Pokemon` and `6 Digimon`. 👍 💻

Reply:
Hello @Gambit142
Thank you for your feedback. I checked it twice and got the following result for `Pokemon` and `Digimon`. Please consider the following result. It shows that there are `4` rows of `Pokemon` and `6` rows of `Digimon`.
**Kindly advise me on this 😄**

![image](https://user-images.githubusercontent.com/106119105/206319094-c6b12b8b-9194-4d64-a4c9-52509033ff56.png)

Reply:
Hello @Gambit142 
I checked it again. In Tuesday's project, there was a query in the requirement to delete one animal. Before that, there were a total of `11` animals. But after the deletion, the number updated to `10`. I am attaching the screenshots of Tuesday's project requirement as well as my solution (which is already approved)

![image](https://user-images.githubusercontent.com/106119105/206427317-28ee4a6b-0e9d-4797-ae3b-75e3e5546fe6.png)
![image](https://user-images.githubusercontent.com/106119105/206427860-a8b411a7-1e44-4f5c-bb50-e8d8d69339f0.png)
![image](https://user-images.githubusercontent.com/106119105/206427969-02995ae2-a084-42ef-a8c1-485b557aac6f.png)

**So, the total number of animals present at a moment is `10`. In this case, `4` rows of Pokemon and `6` rows of Digimon make sense.**

**Kindly advise me on this. Waiting for your reply.**